### DISCUSSION TITLE
Neopixel/RGBW light strip

### DISCUSSION THREAD

**Discussion by caschy** [FundamentalFrequency/.github#28] [General]
Any ideas on how to control an LED light strip?  Does Tracktion engine have any plugins for this?  If not, can the teensy firmware incorporate some sort of control for a strip.  Just looking for random effects at this point unless I guess Tracktion engine can itself control it (via the Pi of course).

**Comment by reklm-e** [comment]
Do want only the "piano" Keys  or all keys to light
?

**Reply by reklm-e** [reply]
you could modify the teensy code to work with neopixl

**Comment by caschy** [comment]
Thanks yes, I'm looking at the code.  Any chance there is a pin on teensy that isn't being used?

**Reply by reklm-e** [reply]
i dont know im not gonna be using the teensy

**Comment by caschy** [comment]
Actually, seems like the control surface plugin already has this functionality built in and compatible wit the teensy.
https://tttapa.github.io/Control-Surface-doc/Doxygen/d8/dd2/9_8Note-FastLED_8ino-example.html

**Reply by pixiesue** [reply]
I can't wait to see what you plan to do with a rgb strip
